autometa/common/coverage.py

```diff
@@ -174,8 +174,8 @@ def make_bed(lengths=lengths, fasta=fasta, bam=bam, bed=bed):
         def sort_samfile(sam=sam, bam=bam, nproc=nproc):
             try:
                 samtools.sort(sam, bam, cpus=nproc)
-            except SamtoolsSortError:
-                raise SamtoolsSortError
+            except ExternalToolError:
+                raise ExternalToolError
```

Comment:
Here we are handling the exception. so we may log it here. Right now you are just raising the error which is not doing anything. At the moment, there is no reason for the try/except block because the same behavior would occur without it. This is the location where we could exit gracefully while logging the error information to the initialized logger.

```suggestion
            except ExternalToolError as err:
                logger.error(err)
                # Not sure if we want to exit in this manner. But something like this...
                sys.exit(-1)
```

@jason-c-kwan may have an opinion on how we should handle exiting Autometa when an error is encountered _and handled_.